pkg/agent/cniserver/pod_configuration_linux.go

```diff
@@ -40,17 +40,13 @@ func (pc *podConfigurator) connectInterfaceToOVS(
 	ovsPortName := hostIface.Name
 	containerConfig := buildContainerConfig(ovsPortName, containerID, podName, podNamespace,
 		netNS, containerIface, ips, vlanID)
-	return containerConfig, pc.connectInterfaceToOVSCommon(ovsPortName, netNS, containerConfig)
-}
 
-func (pc *podConfigurator) connectInterfaceToOVSCommon(ovsPortName, netNS string, containerConfig *interfacestore.InterfaceConfig) error {
-	// create OVS Port and add attach container configuration into external_ids
-	containerID := containerConfig.ContainerID
-	klog.V(2).Infof("Adding OVS port %s for container %s", ovsPortName, containerID)
+	// Create an OVS Port and add container configuration into external_ids.
 	ovsAttachInfo := BuildOVSPortExternalIDs(containerConfig)
+	klog.V(2).Infof("Adding OVS port %s for container %s", ovsPortName, containerID)
```

Comment:
```suggestion
	klog.V(2).InfoS("Adding OVS port for container", "port", ovsPortName, "container", containerID)
```

It will also match the Windows version of this log this way

Reply:
Done